src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmExceptionCode.cs

```diff
@@ -20,1179 +20,8 @@ public enum DkmExceptionCode
         E_FAIL = -2147467259,
         //
         // Summary:
-        //     A debugger is already attached.
-        E_ATTACH_DEBUGGER_ALREADY_ATTACHED = -2147221503,
```

Comment:
Only a few of these values were used, and many were missing. Reduced this to just those values that are used.